Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Home.kt

```diff
@@ -189,13 +187,10 @@ private fun JetsnackBottomNavLayout(
             Animatable(if (i == selectedIndex) 1f else 0f)
         }
     }
-    val scope = rememberCoroutineScope()
     selectionFractions.forEachIndexed { index, selectionFraction ->
         val target = if (index == selectedIndex) 1f else 0f
-        if (selectionFraction.targetValue != target) {
-            scope.launch {
-                selectionFraction.animateTo(target, animSpec)
-            }
+        LaunchedEffect(target) {
```

Comment:
Does this work? I think you need to add the `index` as one of the keys, so that you launch multiple concurrently.

Reply:
It does indeed work
![js_nav](https://user-images.githubusercontent.com/352556/108404823-36da7480-7218-11eb-8971-bf5efe56c219.gif)

Reply:
You might consider using the animSpec as an additional effect key depending on the behavior you want if it changes, finish the current animation in the current spec or change mid-animation.

Reply:
Good call, done.